roles/tests-integration/tests/common/mod.rs

```diff
@@ -0,0 +1,194 @@
+use std::sync::Arc;
+
+use async_std::net::TcpStream;
+use key_utils::{Secp256k1PublicKey, Secp256k1SecretKey};
+
+pub async fn start_job_declarator_server() {
+    let authority_public_key = Secp256k1PublicKey::try_from(
+        "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72".to_string(),
+    )
+    .expect("failed");
+    let authority_secret_key = Secp256k1SecretKey::try_from(
+        "mkDLTBBRxdBv998612qipDYoTK3YUrqLe8uWw7gu3iXbSrn2n".to_string(),
+    )
+    .expect("failed");
+    let cert_validity_sec = 3600;
+    let listen_jd_address = "0.0.0.0:34264".to_string();
+    let core_rpc_url = "http://75.119.150.111".to_string();
```

Comment:
Is this a known core node? If so, we should at least comment on where it comes from.

Reply:
Same as below (line 58). Perhaps having defined `const`s at the top of the file would be appropriate. (probably in a follow on PR with `good first issue`

Reply:
this will likely go away with https://github.com/stratum-mining/stratum/issues/1119